BQ25895M.device.nut

```diff
@@ -0,0 +1,302 @@
+//Registers Addresses
+const BQ25895M_REG00 = 0x00;
+const BQ25895M_REG01 = 0x01;
+const BQ25895M_REG02 = 0x02;
+const BQ25895M_REG03 = 0x03; 
+const BQ25895M_REG04 = 0x04;
+const BQ25895M_REG05 = 0x05;
+const BQ25895M_REG06 = 0x06;
+const BQ25895M_REG07 = 0x07;
+const BQ25895M_REG08 = 0x08;
+const BQ25895M_REG09 = 0x09;
+const BQ25895M_REG0A = 0x0A;
+const BQ25895M_REG0B = 0x0B;
+const BQ25895M_REG0C = 0x0C;
+const BQ25895M_REG0D = 0x0D;
+const BQ25895M_REG0E = 0x0E;
+const BQ25895M_REG0F = 0x0F;
+const BQ25895M_REG10 = 0x10;
+const BQ25895M_REG11 = 0x11;
+const BQ25895M_REG12 = 0x12;
+const BQ25895M_REG13 = 0x13;
+const BQ25895M_REG14 = 0x14;
+
+// For getChargeStatus() output
+enum chargingStatus{
```

Comment:
Style - enums are constant values and so should be all caps

enum BULLWINKLE_MESSAGE_TYPE {
    SEND,
    REPLY,
    ACK,
    NACK,
    TIMEOUT,
    DONE
}